src/components/common/Button.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'proptypes';
+import classNames from 'classnames';
+
+const Button = ({
+  id,
+  label,
+  handleClick,
+  color,
+  light,
+  size,
+  fullWidth,
+  outlined,
+  inverted,
+  rounded,
+  hovered,
+  focused,
+  active,
+  loading,
+  isStatic,
+  disabled,
+}) => {
+  // Dynamically generates button className from props to comply with Bulma styling modifiers.
+  const buttonClassName = classNames('button', {
+    [`is-${color}`]: color,
+    'is-light': light,
+    [`is-${size}`]: size,
+    'is-fullwidth': fullWidth,
+    'is-outlined': outlined,
+    'is-inverted': inverted,
+    'is-rounded': rounded,
+    'is-hovered': hovered,
+    'is-focused': focused,
+    'is-active': active,
+    'is-loading': loading,
+    'is-static': isStatic,
+    'is-disabled': disabled,
+  });
+
+  const buttonId = `btn-${id}`;
+
+  return (
+    <button
+      id={buttonId}
+      type="button"
+      onClick={handleClick}
+      disabled={disabled}
+      className={buttonClassName}
+    >
+      {label}
+    </button>
+  );
+};
+
+export default Button;
+
+Button.propTypes = {
+  id: PropTypes.string,
```

Comment:
required or supply a default value

Reply:
Changed id to a required prop on Button, Checkbox, ToggleSwitch components.